velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -709,14 +713,25 @@ void ExpressionFuzzer::go() {
       resultVector = vectorFuzzer_.fuzzFlat(plan->type());
     }
 
+    // Randomly pick columns from the input row vector to wrap in lazy.
```

Comment:
nit: wrapping these things into smaller helper function tend to make the code more readable:

`auto columnToWrapInLazy = generateLazyColumnIds();`